pydm/widgets/image.py

```diff
@@ -272,7 +272,9 @@ def colorMap(self, new_cmap):
         new_cmap : PyDMColorMap
         """
         self._colormap = new_cmap
-        self._cm_colors = self.color_maps[new_cmap]
+        if new_cmap:
```

Comment:
Why not also wrap the ``self._colormap`` with this ``if`` statement?

Reply:
Do you mean checking for self.color_maps?

if new_cmap and self.color_maps:
...

Reply:
I mean make it:
```
if new_cmap:
    self._colormap = new_cmap
    ...
```
Otherwise self._colormap can be `None`.